_inc/client/components/jetpack-notices/state-notices.jsx

```diff
@@ -232,14 +232,13 @@ class JetpackStateNotices extends React.Component {
 		const currentVersion = this.props.currentVersion;
 		const versionForUpgradeNotice = /(6\.8).*/;
 		const match = currentVersion.match( versionForUpgradeNotice );
-		if ( 'modules_activated' === message && match ) {
+		if ( 'modules_activated' === message && match && true === isGutenbergAvailable ) {
```

Comment:
Also does `isGutenbergAvailable()` return `true` or `1`?

![image](https://user-images.githubusercontent.com/7129409/48868020-312a2600-eda6-11e8-95ab-1b77bce33a2b.png)

You could default the selector to return `false` instead of an empty string, and just check the existence of `this.props.isGutenbergAvailble` without the comparison

Reply:
I believe 84043a6220f90645cd4c8bc231707afe5b4c7431 should take care of this.